src/main/java/com/transportcompany/transportcompany/models/entities/Drivers.java

```diff
@@ -0,0 +1,15 @@
+package com.transportcompany.transportcompany.models.entities;
+import jakarta.persistence.*;
+
+@Entity
+@Table(name = "driver")
+public class Drivers {
```

Comment:
Entities should use the singular, and the table should use the plural, so we would have:

```
@Entity
@Table(name = "drivers")
public class Driver { }
```

Please please don't forget about the code formatting and also to leave an empty row in `each` class, between the name of the class and the first line of code.